src/App.jsx

```diff
@@ -1,17 +1,14 @@
-import './css/App.css';
-import { api } from './api';
-import { Heading, SimpleGrid } from '@chakra-ui/react';
+import "./css/App.css";
+import { Container } from "@chakra-ui/react";
```

Comment:
Mixing multiple libraries - bootstrap vs. Chakra vs. custom CSS - it is better to pick one approach and stick with it. With every library added you are increasing "bundle size" and making the application perform a little bit worse. Everything is a trade-off 🤷‍♂️